docs/buildah-build.1.md

```diff
@@ -476,6 +476,10 @@ If the specified capabilities are not in the default set, container engines
 should print an error message and will run the container with the default
 capabilities.
 
+**--ltoa** *bool-value*
```

Comment:
I find the `ltoa` name very confusing. In my programmer brain this means "long to ascii" which is also very close to the more common `itoa`.
I prefer names that a normal person can understand without looking up the man page.

Reply:
@Luap99 , How does `lta` or `l-to-a` or `l2a` grab you?  Although "el" in front of 2 might be mistaken for the word "one" instead of "el".